### PR TITLE
Cleanup & Docker

### PR DESCRIPTION
**Pull request by rschlaefli**
### Description

- Improve and refactor Docker related files for consistency
- Package the preprocessing script in an init-service
- Upgrade to NodeJS 14
- Upgrade all dependecies to the latest version

### Related Issues

- https://github.com/kwizapp/kwiz/issues/39

### Checklist

- [x] Have you added tests where necessary? Do all the test pass? 
- [x] Have you added descriptive comments to your code?
- [ ] Have you updated the documentation related to this propo

### CODE REVIEW

File: preprocessing/run.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/wait-for.sh $DATABASE_HOSTNAME:5432 -- python -u /app/Preprocessing.py
```
Comment (alexscheitlin) [inline preprocessing/run.sh:3]: I'm not able to execute the `run.sh` script as there is no `wait-for.sh` script in my root folder.
Is there some dependency that needs to be installed or is it only thought for the build or the docker-compose from the umbrella repo?

**Review by alexscheitlin — CHANGES_REQUESTED** [review]

File: preprocessing/run.sh (hunk 1 above)
Comment (rschlaefli) [reply preprocessing/run.sh:3]: that script is only embedded in the docker image and the run script is used in there as well ;) no need to call it manually

Comment (rschlaefli) [reply preprocessing/run.sh:3]: docker-compose will execute the init container

Comment (alexscheitlin) [reply preprocessing/run.sh:3]: Allright, thanks for the clarification (Y)

**Review by alexscheitlin — APPROVED** [review]

**Pull request merged**